targets/ipq40xx-generic

```diff
@@ -1,18 +1,18 @@
 local ATH10K_PACKAGES_IPQ40XX = {
-	'kmod-ath10k',
-	'-kmod-ath10k-ct',
+	'-kmod-ath10k',
+	'kmod-ath10k-ct',
 	'-kmod-ath10k-ct-smallbuffers',
-	'ath10k-firmware-qca4019',
-	'-ath10k-firmware-qca4019-ct',
+	'-ath10k-firmware-qca4019',
+	'ath10k-firmware-qca4019-ct',
 }
 local ATH10K_PACKAGES_IPQ40XX_QCA9888 = {
-	'kmod-ath10k',
-	'-kmod-ath10k-ct',
+	'-kmod-ath10k',
+	'kmod-ath10k-ct',
 	'-kmod-ath10k-ct-smallbuffers',
-	'ath10k-firmware-qca4019',
-	'-ath10k-firmware-qca4019-ct',
-	'ath10k-firmware-qca9888',
-	'-ath10k-firmware-qca9888-ct',
+	'-ath10k-firmware-qca4019',
+	'ath10k-firmware-qca4019-ct',
+	'-ath10k-firmware-qca9888',
+	'ath10k-firmware-qca9888-ct',
```

Comment:
I think we should just drop these variables altogether where they don't change anything.

Reply:
Thought the same, but i won't rule out we need to change it in the future, so I would keep them.

Reply:
Hmm, or keep the variable, but make it an empty list? That would also give us the smallbuffers configuration where OpenWrt defaults to it.

Reply:
Empty list sounds like a good compromise.